itou/siaes/admin.py

```diff
@@ -26,7 +26,7 @@ class JobsInline(admin.TabularInline):
     model = models.Siae.jobs.through
     extra = 1
     raw_id_fields = ("appellation", "siae", "location")
-    readonly_fields = ("created_at", "updated_at")
+    readonly_fields = ("created_at", "updated_at", "location")
```

Comment:
hmmm on peut quand meme la changer du coup ? ou faut aller sur le Job c'est ça ?

Reply:
Yes, mais du coup moi ça me titilllerait presque un jour de pouvoir carrément juste mettre un lien vers des pages de fiche de poste et dégager tout l'inline ! Mais bon oui, on s'en fiche. Débloquons. Tu peux mettre un commentaire qui explicite cela ? Si un dev dans 6 mois se demande genre "tiens c'est bizarre on devrait pouvoir modifier la `location`" et retire ça des readonlyfields, patatras !

Reply:
Et après, LGTM of course

Reply:
Ok je mets un commentaire puis MEP.

Reply:
Encore `location` ...

Reply:
En fait c'est bizarre on a bien une page admin des fiches de poste ex http://localhost:8080/admin/siaes/siaejobdescription/25254/change/ mais bizarrement dans la fiche SIAE, les fiches de poste en inline pointent vers les fiches appelations et pas les fiches de poste... alors que les membres en inline pointent bien vers les fiches users 🤔